doc/source/entity-sample-definitions.rst

```diff
@@ -0,0 +1,194 @@
+.. _entity-sample-definitions:
+
+======================
+Example entity samples
+======================
+
+A sample represents an entity at a given point in time.  There are a few different ways to define an entity using the following fields: 
+ **id**
+     a unique string that represents a specific object that exists.
+ **kind**
+     the type or category of the object.
+ **time**
+     A :py:class:`datetime.datetime` at which the entity state described by this sample is valid.
+
+And one of the following location fields:
+ **point**
+     A coordinate pair that defines a location in a 2D cartisian coordinate system.
+ **path**
+     A list of coordinates that define a series of connected line segments in a 2D cartisian coordinate system.
+ **polygon**
+     A list of coordinates that define a closed shape in a 2D cartisian coordinate system.
+
+These options provide the user with the means to define entities with three different spatial representations and convenient structures for defining geographic coordinates.
+
+++++++++++++++++++++
+Locations
+++++++++++++++++++++
+
+Location fields are used to define where an entity exists and the entity's shape.
+
+--------------------
+Point sample
+--------------------
+
+A point sample defines an entity as a point in space.  Point samples are best used to represent entities in substrates other than the world map.::
+
+    {
+        "id": 1,
+        "kind": "ball",
+        "time": datetime.datetime.now()
+        "point": {
+            "x": 571045,
+            "y": 22131
+        }
+    }
+
+------------------------
+Geographic point sample
+------------------------
+
+A geopoint sample defines an entity at a point in space.  Coordinates are defined in latitude and longitude for convenience.::
+
+    {
+        "id": 1,
+        "kind": "truck",
+        "time": datetime.datetime.now()
+        "point": {
+            "lon": -91.571045,
+            "lat": 38.022131
+        },
+    }
+
+A geographic sample could also be represented with a point sample.::
```

Comment:
While it is implied given the example, it may be worth explicitly stating that x maps to longitude and y maps to latitude